STM32F405RGTx/Core/Src/rtos_threads/measurements_thread.cpp

```diff
@@ -89,6 +89,7 @@ void MeasurementsThread::startADCandDMA() {
     
     if (status != HAL_OK) {
         printf("Error: HAL_ADC_Start_DMA failed with status code %d\r\n", status);
```

Comment:
You can remove this printf. Generally, we're not going to be connected over UART so there's no reason to have it in. If we're debugging over UART, you can just add prints as you need.